base-theme/layouts/partials/resource_layout.html

```diff
@@ -0,0 +1,30 @@
+<div class="resource-page container">
+  {{- if ne .Params.resourcetype "Video" }}
+      {{- if .Content -}}
+          <div class="row">
+              <div class="label col-3">Description:</div>
+              <div class="col-9">{{ .Content }}</div>
+          </div>
+      {{- end -}}
+      {{- if .Params.learning_resource_types -}}
+          <div class="row">
+              <div class="label col-3">Resource Type:</div>
+              <div class="col-9">{{ delimit (.Params.learning_resource_types | default slice) ", " }}</div>
+          </div>
+      {{- end -}}
+      {{- if .Params.file -}}
+      <div class="row">
+          <div class="col-12 d-flex flex-direction-row align-items-center">
+              <a class="download-file" href="{{ partial "resource_url" .Params.file }}"><span class="material-icons">file_download</span> Download File</a>
+          </div>
+      </div>
+      {{- end -}}
+  {{- else -}}
+      {{ partial "video.html" . }}
```

Comment:
So I tested video resources with this and while the partials for the video resource page have been moved over, there seems to be an issue with the video metadata.  The [starter configuration](https://github.com/mitodl/ocw-hugo-projects/blob/main/ocw-www/ocw-studio.yaml) for `ocw-www` doesn't have all the necessary video metadata that the `ocw-course` starter has, so that will need to be added.  There may also be some changes necessary in `ocw-studio` relating to the processing of uploaded videos, but I'm not certain.  This is outside the scope of this PR, but this will need to be addressed if we want videos to work in `ocw-www`.

Reply:
We can leave videos for later. What we've been waiting for, for a while, is PDFs.